### ISSUE TITLE
Update Blog styles

### ISSUE THREAD

**Issue by nickwynja** [infor-design/website#537]
From #336:

***
@kayiuho and @jmacaluso711 Hey, sorry chiming in late here. A couple of thoughts on the landing page

- I wonder if we need the yellow bar up top. I think its the placement thats throwing me off. Since its full width under the header it feels more like an alert banner as opposed to just a visual design element.

- Blog post truncation. Looks like the entire blog is posted on the landing page with the h1 being a link. I wonder if we should truncate the body content to like 170 characters (about a line and half). We might have really long blog posts as the content expands and having the entire article would increase scroll time.

- The newsletter callout on the bottom feels maybe a bit too large. The type hierarchy feels like its competing with the content above it, as opposed to being secondary content. 

what if we..
- [ ] use h2 bold and p fonts
- [ ] center align items in div to max width of the body above
since the body above is 5 cols, and the NL is left aligned at 7 body col there is a bit of a jump when reading left to right
![screen shot 2018-05-03 at 2 44 27 pm](https://user-images.githubusercontent.com/34688242/39596489-87d338bc-4ee0-11e8-839c-c5ea5b7355ac.png)

- [ ] flex input form and buttons on one line 
- [ ] make input forms smaller
- [ ] make input background white
- [ ] update copy on button to be "Subscribe" feels more explicit
- [ ] maybe a different color? the yellow feels like a status color 

![screen shot 2018-05-03 at 2 29 44 pm](https://user-images.githubusercontent.com/34688242/39595844-9d0930d0-4ede-11e8-972e-d8b0dd1c3804.png)

**Comment by kayiuho** [comment]
@kmahsmith those are all great feedback! I can tweak the blog design for sure. There's another ticket out for a new footer, which will have a NL hook. And because of that, we may not need the bottom "subscribe" banner on the blog page.. I'll play around with that.

**Comment by theakatz** [comment]
footer style dependent on #508

**Comment by kayiuho** [comment]
Since we've wrapped up the footer, the only 2 changes I see to do here is
- [ ] remove top yellow bar
- [ ] truncating the body copy on the blog feed to 170 characters

@jmacaluso711 can you update?
